### ISSUE TITLE
Multi target prediction

### ISSUE THREAD

**Issue by MichaelMedek** [dholzmueller/pytabkit#17]
Is there a way to predict multiple targets at once? Currently X is a DataFrame and y is a Series. Sklearn also supports y as DataFrame with e.g. 2 columns for some models. Especially MLPs could scale well if they are trained with 2 targets simultaneously, since they might depend on each other and it would make things faster if one is interested in more targets at once.

**Comment by dholzmueller** [comment]
Mixed classification and regression is currently not supported, neither is multi-label classification (and I don't intend on adding it, as it would complicate things a bit, unfortunately...). My comment was solely for multi-output regression.

**Comment by dholzmueller** [comment]
Interesting! Yeah I think there is much potential to popularize NNs on tabular data by having some proper software development team giving them all these features (custom loss functions, sample weights, etc.). Unfortunately I'm not in a position to do all of this at the moment.
